data/ui/AboutDialog.ui.in

```diff
@@ -15,8 +15,21 @@
     Jente Hidskes &lt;hjdskes@gmail.com&gt;
     Peter Hutterer &lt;peter.hutterer@who-t.net&gt;
 
+    <!--  Contributor list: git log | git shortlog \-\-summary | cut -f2 -->
 Contributors:
-	Dan Callaghan &lt;djc@djc.id.au&gt;
+        AsavarTzeth
+        Benjamin Tissoires
+        Dan Callaghan
+        FFY00
+        Ikey Doherty
+        Jente Hidskes
```

Comment:
I'm already under authors, should I be under contributors too?

Reply:
eh, it's just the git shortlog. Figured leaving it as-is is easier than thinking about it :)

Reply:
Alrighty, let's merge this then!